client/blocks/importer/wordpress/import-everything/pre-migration/test/index.test.tsx

```diff
@@ -82,24 +101,29 @@ function renderPreMigrationScreen( props?: any ) {
 }
 
 describe( 'PreMigration', () => {
+	beforeAll( () => {
+		nock.disableNetConnect();
+	} );
+
 	beforeEach( () => {
 		jest.clearAllMocks();
 	} );
 
-	test( 'should show Upgrade plan screen', () => {
-		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-		// @ts-ignore
-		useSiteMigrateInfo.mockReturnValue( {
+	test( 'should show Upgrade plan screen', async () => {
+		( useSiteMigrateInfo as jest.Mock ).mockReturnValue( {
 			sourceSiteId: 777712,
 			fetchMigrationEnabledStatus: jest.fn(),
 			isFetchingData: false,
 			siteCanMigrate: true,
 			isInitFetchingDone: true,
 		} );
 
-		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-		// @ts-ignore
-		useCheckEligibilityMigrationTrialPlan.mockReturnValue( {
+		( useUpgradePlanHostingDetailsList as jest.Mock ).mockReturnValue( {
```

Comment:
Since I had this change here, I just changed the other mocks to also use the `as jest.Mock`, similar to other parts of the codebase, so we don't need to add the lint ignore.